CHANGELOG.md

```diff
@@ -1,5 +1,27 @@
 See the [releases](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/releases) for details on bug fixes and added features.
 
+8.2.0
+=====
+### Fundamentals
+- Update System.Text.Json to 8.0.5 CVE-2024-43485. See [2892](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/2892).
+- Using FixedTimeEquals in NETCore targets. See [2857](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/2857).
+- Updated .NET 9 to RC 2 [2898](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/2898).
+- Adds ability to create token without kid [2968](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/2968)
+- Enables code coverage in PRs [2946](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/2946)
+- Various test improvements:
+- (#2953)[https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/2953]
```

Comment:
```suggestion
- #2953
```
I think we can do this now 🤔 
Adding all the links takes a lot to work (and fixing all the ones below), might be worth testing if this works.

Reply:
Done. Thank you for pointing it out Kelly.